docs/src/reference/asciidoc/core/errorhandlers.adoc

```diff
@@ -345,3 +345,292 @@ es.write.rest.error.handler.writeFile.filename = /path/to/some/output/file
 You now have a chain of handlers that retries bulk rejections by default (HTTP Retry built-in handler), then ignores
 any errors that are conflicts (our own ignore conflicts handler), then ignores any other errors by writing them out to
 a file (our own output to file handler).
+
+[[errorhandlers-read-json]]
```

Comment:
Just confirming here, this will be a separate page, right?

Reply:
Correct.